src/visor/readable.go

```diff
@@ -145,10 +147,12 @@ func NewReadableTransactionOutput(t *coin.TransactionOutput, txid cipher.SHA256)
 }
 
 // NewReadableTransactionInput creates readable transaction input
-func NewReadableTransactionInput(uxID string, ownerAddress string) ReadableTransactionInput {
+func NewReadableTransactionInput(uxID string, ownerAddress string, coins string, hours string) ReadableTransactionInput {
```

Comment:
This function signature can be rewritten as `(uxID, ownerAddress, coins, hours string)`